src/ir_Hitachi.h

```diff
@@ -11,7 +11,8 @@
 //   Brand: Hitachi,  Model: PC-LH3B (HITACHI_AC3)
 //   Brand: Hitachi,  Model: KAZE-312KSDP A/C (HITACHI_AC1)
 //   Brand: Hitachi,  Model: R-LT0541-HTA/Y.K.1.1-1 V2.3 remote (HITACHI_AC1)
-//   Brand: Hitachi,  Model: RAS-22NK remote (HITACHI_AC344)
+//   Brand: Hitachi,  Model: RAS-22NK A/C (HITACHI_AC344)
+//   Brand: Hitachi,  Model: RF11T1 remote (HITACHI_AC344)
```

Comment:
Not an issue, just a question
There is some data about support in the `.cpp` file as well,
Should that data be removed/moved, and the support data should only be in the `.h` file?

Also seen some support data that does not match the new format, want to change that? or are they left as is because support are unsure?
Should `SupportedProtocols.md` be regenerated?

Reply:
IIRC, the tool scrapes both *.h & *.cpp files to build up it's picture.
The _general_ rule of thumb is "store it in the .h file if it exists.".

> Should that data be removed/moved, and the support data should only be in the .h file?

Probably. Feel free to.

> Also seen some support data that does not match the new format, want to change that? or are they left as is because support are unsure?

Probably. Feel free to. I did a quick pass over the code when we (@sillyfrog & I) added this sort of tool. I probably missed things.

> Should SupportedProtocols.md be regenerated?

I was going with your suggestion of a while back, of really only running at when we do a release. There are good reasons for and against running it out-of-release-cycle. I'm now on the fence after your discussion last time. ;-)

Reply:
So I would then like to propose (and will try to implement):
* if there is a .h file, warn if there is anything in the cpp file
* try to search for old format data, and warn about it

Good on the generation part, just want to be consistent.

Reply:
SGTM